doc/content/sqa/malamute_sdd.md

```diff
@@ -18,7 +18,9 @@ with no additional dependencies.
 !template-end!
 
 !template! item key=design-stakeholders
-!include framework_sdd.md start=design-stakeholders-begin end=design-stakeholders-finish
+Stakeholders for {{app}} include several of the funding sources including [!ac](DOE) and [!ac](INL).
+However, Since {{app}} is an open-source project, several universities, companies, and foreign governments
```

Comment:
do we have all of these entities active in MALAMUTE? or are we planning ahead?

Reply:
DOE/INL paid for initial MALAMUTE development, so those are the main stakeholders. We've had users from universities, and interest from companies. Foreign governments is a bit of planning ahead (and consistent with the boilerplate language from MOOSE).

Reply:
sounds god to me, thanks!